iothub_client/inc/iothub_device_client_ll.h

```diff
@@ -378,6 +378,88 @@ typedef struct IOTHUB_CLIENT_CORE_LL_HANDLE_DATA_TAG* IOTHUB_DEVICE_CLIENT_LL_HA
      */
      MOCKABLE_FUNCTION(, IOTHUB_CLIENT_RESULT, IoTHubDeviceClient_LL_UploadMultipleBlocksToBlob, IOTHUB_DEVICE_CLIENT_LL_HANDLE, iotHubClientHandle, const char*, destinationFileName, IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_CALLBACK_EX, getDataCallbackEx, void*, context);
 
+     /**
+     * @brief    This API creates a context for a new blob upload to Azure Storage.
+     * @remark   It is part of a set of functions for more granular control over Azure IoT-based blob uploads.
```

Comment:
Instead of pasting this everywhere, please write a separate article / documentation for the object (i.e. `IOTHUB_CLIENT_LL_UPLOADTOBLOB_CONTEXT_HANDLE`).